### PR TITLE
Update to govuk-frontend v5.0.0

### PR DESCRIPTION
**Pull request by DavidBiddle**
### What problem does this pull request solve?

Trello card: https://trello.com/c/VeOkfpZv/1250-update-forms-runner-to-govuk-frontend-v5

Updates the govuk-frontend npm package and the govuk_components and govuk_design_system_formbuilder gems to v5.0.0.

Also includes several changes mandated or suggested by the upgrade guide - see individual commit messages for details. 

### Screenshots
Visually things are almost the same, except tags are now sentence-case rather than uppercase:
The header of a form, showing a pink tag reading 'Local' between the GOV.UK logo and the form name. Below this is a banner containing a blue 'Live preview' tag next to the text 'You're previewing the live version of this form.

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

**Review by thomasiles — APPROVED** [review]
Amazing! Really nicely structured and easy to follow.

looks good to me and works locally and in docker 🏅

File: app/views/layouts/base.html.erb [hunk 1]
```diff
@@ -8,27 +8,19 @@
...
-    <link rel="apple-touch-icon" href="<%= vite_asset_path "#{govuk_assets_path}/images/govuk-apple-touch-icon-180x180.png"%>" sizes="180x180" />
-    <link rel="apple-touch-icon" href="<%= vite_asset_path "#{govuk_assets_path}/images/govuk-apple-touch-icon-167x167.png"%>" sizes="167x167" />
-    <link rel="apple-touch-icon" href="<%= vite_asset_path "#{govuk_assets_path}/images/govuk-apple-touch-icon-152x152.png"%>" sizes="152x152" />
-    <link rel="apple-touch-icon" href="<%= vite_asset_path "#{govuk_assets_path}/images/govuk-apple-touch-icon.png"%>" />
+    <link rel="icon" sizes="48x48" href="<%= vite_asset_path "#{govuk_assets_path}/images/favicon.ico" %>">
+    <link rel="icon" sizes="any" href="<%= vite_asset_path "#{govuk_assets_path}/images/favicon.svg" %>" type="image/svg+xml">
+    <link rel="mask-icon" href="<%= vite_asset_path "#{govuk_assets_path}/images/govuk-icon-mask.svg" %>" color="#0b0c0c">
+    <link rel="apple-touch-icon" href="<%= vite_asset_path "#{govuk_assets_path}/images/govuk-icon-180.png"%>">
+    <link rel="manifest" href="<%= vite_asset_path "#{govuk_assets_path}/manifest.json" %>">
```
Comment (lfdebrux) [inline app/views/layouts/base.html.erb:15]: Does our asset pipeline do fingerprinting? If yes, we might need to generate our own manifest file (see https://github.com/alphagov/govuk-frontend/discussions/4389#discussioncomment-7697959).

Comment (DavidBiddle) [reply app/views/layouts/base.html.erb]: Good spot - we do fingerprint so you're right, the manifest won't work. For now I've moved the manifest and the icons into the public folder to sidestep the issue, but I'd like to have a look at either copying them automatically as part of the build process, or generating the manifest (possibly with [vite-plugin-pwa](https://github.com/vite-pwa/vite-plugin-pwa)).

Comment (lfdebrux) [reply app/views/layouts/base.html.erb]: Makes sense, thanks for looking into that.

Might be worth adding a `<!-- TODO -->` comment to this file to say something about the copies of the images?

Comment (DavidBiddle) [reply app/views/layouts/base.html.erb]: Good suggestion, have updated

**Review by lfdebrux — APPROVED** [review]
Nice one!

**Pull request merged**